client/src/telemetry/constants.ts

```diff
@@ -29,6 +29,8 @@ export const SIDEBAR_CLICK_WITH_FILTER = "sidebar_click_with_filter";
 export const SIDEBAR_FILTER_FOCUS = "sidebar_filter_focus";
 export const TOP_NAV_ALREADY_SUBSCRIBER = "top_nav_already_subscriber";
 export const TOP_NAV_GET_MDN_PLUS = "top_nav_get_mdn_plus";
```

Comment:
Should these be removed as they're no longer used?

Reply:
Kept them around, as we still have Glean data until ~ the end of year that we might use for before/after comparison. But I marked them as deprecated now.